Bottom: 8ee1fef2fb4b5fd3fb42f74ad3c7e3b8d1ca5182
Top:    9bc8f3beab6508ae50e94aaf6e40bb739cb35771
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-21 19:05:23 +0000

Re-enable soldermask emission


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index efe337c..8d81a8e 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1040,7 +1040,7 @@ object3d_from_soldermask_within_area (POLYAREA *area, int side)
   LayerType *layer;
   POLYAREA *pa;
 
-  return NULL;
+//  return NULL;
 
   poly_Copy0 (&info.poly, area);
   info.side = side;
